packages/config/src/chainConfig/types.ts

```diff
@@ -1,10 +1,11 @@
-/* eslint-disable @typescript-eslint/naming-convention */
 import {PresetName} from "@chainsafe/lodestar-params";
 
+/* eslint-disable @typescript-eslint/naming-convention */
+
 /**
  * Run-time chain configuration
  */
-export interface IChainConfig {
+export type IChainConfig = {
```

Comment:
why change to type?

Reply:
Otherwise Typescript doesnt allow this to be used as a Record<string,unknown> argument in functions